osu.Server.Spectator/Database/DatabaseAccess.cs

```diff
@@ -67,11 +67,6 @@ public async Task<multiplayer_room> GetRoomAsync(long roomId)
             });
         }
 
-        public async Task<string?> GetBeatmapChecksumAsync(int beatmapId)
-        {
-            return (await GetBeatmapAsync(beatmapId))?.checksum;
-        }
```

Comment:
This method is extensively mocked in unit tests, so removing it and not adjusting the tests that mock `GetBeatmapChecksumAsync()` to mock `GetBeatmapAsync()` instead makes the unit tests not compile and also not pass.

Reply:
I replaced all uses of `GetBeatmapChecksumAsync()`.

Reply:
You missed one, but I fixed it for you in ed21209d1f8a650c38d76d2c9695e19745a76d0c.